tests/TestCase.php

```diff
@@ -25,4 +25,14 @@
 abstract class TestCase extends BaseTestCase
 {
     use ExpectDeprecationTrait;
+
+    final public function testNotDefiningConstructor(): void
+    {
+        $reflection = new \ReflectionObject($this);
+
+        self::assertNotSame(
```

Comment:
@kubawerlos 
whenever doing things like this, use the message parameter. JohnDoe will see that they done sth we didn't want, but without understanding "why" we don't want it.

Reply:
I believe default messages for each assert are good enough

Reply:
huh? @Wirone , once more my msg:

> JohnDoe will see that they done sth we didn't want, but **without understanding** "why" we don't want it.

So let say JohnDoe wanted to contribute for first time, they did new test and use constructor in it.
Then, they get this result:
```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ ./vendor/bin/phpunit tests/SomeTest.php 
PHPUnit 9.6.15 by Sebastian Bergmann and contributors.

Testing PhpCsFixer\Tests\SomeTest
....F                                                                                                                                                           5 / 5 (100%)

Time: 00:00.045, Memory: 26.50 MB

There was 1 failure:

1) PhpCsFixer\Tests\SomeTest::testNotDefiningConstructor
Failed asserting that two strings are not identical.

/Users/d.ruminski/github/FriendsOfPHP/PHP-CS-Fixer/tests/TestCase.php:33
```


So, JohnDoe got the msg what is failing (`testNotDefiningConstructor`), yet without knowledge **WHY** not do define that constructor.